seed/python-sdk/accept-header/README.md

```diff
@@ -26,7 +26,7 @@ client = SeedAccept(
     token="YOUR_TOKEN",
     base_url="https://yourhost.com/path/to/api",
 )
-client.service.endpoint()
+client.service.with_raw_response.endpoint()
```

Comment:
I'm not sure that we want to update all of the snippets to use the `with_raw_response` alternative. I'd expect none of the `snippet.json`, `reference.md`, or `README.md` examples (like this one) to be updated.

Reply:
Removed all of these changes.